weaver/processes/builtin/__init__.py

```diff
@@ -208,7 +208,7 @@ def run(self, runtime_context, **kwargs):
         try:
             self._validate_process()
             self._update_command()
-            super(BuiltinProcessJobBase, self).run(runtime_context, **kwargs)
+            super(BuiltinProcessJobBase, self).process(runtime_context, **kwargs)
```

Comment:
Not sure about this change. Looks like an error. It should refer to the CWL's run method I believe.